bin/configurer/utils/idam-authenticate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -eu
+
+USERNAME=$1
+PASSWORD=$2
+
+curl --silent --show-error --header 'Content-Type: application/x-www-form-urlencoded' --header 'Accept: application/json' -d "username=${USERNAME}&password=${PASSWORD}" "${IDAM_API_BASE_URL:-http://localhost:5000}/loginUser" | docker run --rm --interactive stedolan/jq -r .api_auth_token
```

Comment:
Isn't that similar / simpler version of `idam-lease-user-token.sh`? I wonder whether we could use that one over `idam-lease-user-token.sh`?

Reply:
Ok let me test that!

Reply:
Unfortunately I was not able to use it, I have asked for support IDAM guys but using the token obtained from the user login gave me 403. 

However the oath2/authorize is deprecated so we should use probably something else, but it doesn't seem loginUser is way to go...